draft-ietf-tls-tls13.md

```diff
@@ -4023,32 +4030,35 @@ and renegotiation_info {{RFC5746}}.
 
 ## Cipher Suites
 
-A symmetric cipher suite defines the pair of the AEAD cipher and hash
-function to be used with HKDF.
+A symmetric cipher suite defines the pair of the AEAD algorithm and hash
+algorithm to be used with HKDF.
 Cipher suite names follow the naming convention:
 
 ~~~
-   CipherSuite TLS_CIPHER_HASH = VALUE;
+   CipherSuite TLS_AEAD_HASH = VALUE;
 ~~~
 
 | Component | Contents |
 |:----------|:---------|
-| TLS     | The string "TLS" |
-| CIPHER    | The symmetric cipher used for record protection |
+| TLS       | The string "TLS" |
+| AEAD      | The AEAD algorithm used for record protection |
```

Comment:
The other changes sound good, but I think all the stuff in this section should stay as-is. A cipher suite is a suite of stuff for a cipher. There's also no real need to de-generalize it here, especially when the term "cipher suite" is well established. I think changing this just risks making it a bit more confusing.